tests/validation/tests/v3_api/common.py

```diff
@@ -1008,7 +1015,7 @@ def validate_dns_record(pod, record, expected, port=TEST_IMAGE_PORT):
         record["name"], record["namespaceId"])
     validate_dns_entry(pod, host, expected, port=port)
 
-def retry_dig(host, pod, expected, retry_count=3):
+def retry_dig(host, pod, expected, retry_count=5):
```

Comment:
Same as prev comment. Any reason this is a count of 5? Can we not use any existing timeouts defined in this python file here?